Mobile/ShopLens/Droid/Helpers/ShopLensSpeechRecognizer.cs

```diff
@@ -6,26 +6,37 @@ namespace ShopLens.Droid.Helpers
 {
     public class ShopLensSpeechRecognizer : ISpeechRecognizer
     {
+        public bool IsListening
+        {
+            get { return voiceListener != null ? true : false; }
```

Comment:
:thinking: :thinking: :thinking: 

:rofl: 

This isn't a rare thing to see, but it's really funny when you think about it

I'm sorry, but I'll leave this one for you to figure out :wink: 

Hint: `voiceListener != null ? true : false`

Reply:
Oh, I could just do "return voiceListener != null", I see.